packages/__examples__/.storybook/stories/TooltipPositioning.tsx

```diff
@@ -0,0 +1,213 @@
+    />,
+    <Link key={19} href="https://instructure.github.io/instructure-ui/">
+      Link
+    </Link>,
+    <List key={20} margin="0 0 small">
+      <List.Item>List item 1</List.Item>
+      <List.Item>List item 2</List.Item>
+    </List>,
+    <Menu key={20} trigger={<Button>Menu</Button>} />,
+    <Metric
+      key={21}
+      textAlign="start"
+      renderLabel="Metric"
+      renderValue="80%"
+    />,
+    <MetricGroup key={22}>
+      <Metric renderLabel="MetricGroup" renderValue="80%" />
+      <Metric renderLabel="Late" renderValue="4" />
+      <Metric renderLabel="Missing" renderValue="2" />
+    </MetricGroup>,
+    <NumberInput key={23} renderLabel="NumberInput" />,
+    <Pill key={24} color="info">
+      Pill
+    </Pill>,
+    <ProgressBar
+      key={25}
+      size="small"
+      screenReaderLabel="progressBar"
+      valueNow={40}
+      valueMax={60}
+      margin="0 0 small"
+    />,
+    <ProgressCircle
+      key={26}
+      size="small"
+      screenReaderLabel="ProgressCircle"
+      valueNow={40}
+      valueMax={60}
+      margin="0 small 0 0"
+    />,
+    <RadioInput key={27} label="RadioInput" value="foo" name="bar" />,
+    <RadioInputGroup
+      key={28}
+      name="example1"
+      defaultValue="val1"
+      description="RadioInputGroup"
+    >
+      <RadioInput key={281} value="val1" label="val1" />
+      <RadioInput key={282} value="val2" label="val2" />
+    </RadioInputGroup>,
+    <Rating
+      key={29}
+      label="Overall rating of freshman year experience"
+      valueNow={68.45}
+      valueMax={100}
+    />,
+    <SimpleSelect key={30} renderLabel="Uncontrolled Select">
+      <SimpleSelect.Option id="simple_Select" value="SimpleSelect">
+        SimpleSelect
+      </SimpleSelect.Option>
+      <SimpleSelect.Option id="baz" value="baz">
+        Baz
+      </SimpleSelect.Option>
+    </SimpleSelect>,
+    <Spinner key={31} renderTitle="Spinner" size="x-small" />,
+    <Tag key={32} text="Tag" margin="0 x-small 0 0" />,
+    <Text key={33}>Text component</Text>,
+    <TextArea key={34} label="This is a TextArea" />,
+    <TextInput key={35} renderLabel="This is a TextInput" />,
+    <View key={36}>This is a View</View>
+  ]
+  return (
+    <ApplyLocale locale="en-US">
+      <br />
+      {components.map(function (component, key) {
+        return wrapComponent(component, key.toString())
```

Comment:
I would suggest wrapping the examples in a `<div key={key}>`, rather than manually setting the keys on the components in the array, because it will be a pain to modify the array like this :D

Edit: as I see you've already done that in the wrapComponent function, are those keys still needed then?